tests/cbmc/proofs/s2n_stuffer_printf/s2n_stuffer_printf_harness.c

```diff
@@ -21,6 +21,17 @@
 #include "stuffer/s2n_stuffer.h"
 #include "utils/s2n_mem.h"
 
+int nondet_int(void);
+
+int vsnprintf(char *str, size_t size, const char *fmt, va_list ap)
+{
+  if(size > 0)
+    __CPROVER_havoc_slice(str, size);
+  (void)*fmt;
+  (void)*(char **)ap;
+  return nondet_int();
```

Comment:
Your indentation doesn't match the rest of the file

Reply:
Fixed.